### ISSUE TITLE
Duplicate definition

### ISSUE THREAD

**Issue by fxjonas** [samtools/samtools#1067]
Dear Samtools team,

I am currently trying to quantify the number of PCR duplicates in our sequencing data. When running mark duplicates on paired-end sequencing, I noticed that it actually doesn't differ between R1 and R2, e.g. if the same DNA fragment is spanned it is counted no matter if the R1 is on Watson or on Crick. However, we only amplify after adapter attachment and a PCR duplicate should thus also have R1 in the same direction as its duplicate. Is there any way to change the duplicate definition in this respect or do you plan to incorporate something like this in the future?

Thanks,
Felix

P.S., for now, my best workaround is creating to separate alignment files (depending on if R1 is on Watson or on crick) and then mark the duplicates separately before fusing both files. If you have a more straight forward solution I would be grateful.

**Comment by daviesrob** [comment]
This should be possible, but we would probably want to make it an option.

**Comment by PedalheadPHX** [comment]
I agree a PCR duplicate is the same strand but isn't this still likely a duplicate in the sense that you don't want to over-represent a single sequence.  I'm thinking about a cancer situation where a mutation in a single cell would lead to an incorrect estimate of the allele frequency in the population when the two strands are sequenced for the mutant molecule but not other alleles.  The only way to truly manage this would be with duplex style adaptors where you could determine if the watson and crick molecules were the same original molecule or not.  But in that case you would do a more advanced error correction beyond just marking duplicate reads.

Also, if you have a PCR error in the first PCR cycle on one of the strands and you keep data for both strands you no longer have a 50/50 chance of carrying the error forward as you would always carry that molecule forward.  Last since so many PCR errors are sequence context dependent and the 5'-3' sequence context is different in the order encountered by the polymerase then you would run the risk of carrying forward PCR errors occurring on one strand, a variant caller should mitigate this but it would be a risky change.

In short if added I'd highly suggest it was only optional and not on by default.  I'd like to see support for UMI tags before something of this nature

`
xx-R1-primingSite--t.....................................A--R2-primingSite-xx (Watson)
xx-R2-primingSite--A....................................t--R1-primingSite-xx (Crick)
  |
PCR
  |
xx-R1-primingSite--t.....................................A--R2-primingSite-xx (Watson)
--------------------------------*-----------------------------<xx (Watson - Duplicate/PCR error)

xx>------------------------------------------------------------- (Crick - Duplicate)
xx-R2-primingSite--A....................................t--R1-primingSite-xx (Crick)
`
